darts/tests/test_timeseries.py

```diff
@@ -1452,3 +1453,113 @@ def test_time_col_convert_garbage(self):
 
         with self.assertRaises(AttributeError):
             TimeSeries.from_dataframe(df=df, time_col="Time")
+
+
+class SimpleStatisticsTestCase(DartsBaseTestClass):
+
+    times = pd.date_range("20130101", "20130110", freq="D")
+    values = np.random.rand(10, 2, 1000)
```

Comment:
(Just in case the tests are a little slow, as there are many of them):
```suggestion
    values = np.random.rand(10, 2, 100)
```